stable/artifactory-ha/templates/artifactory-node-statefulset.yaml

```diff
@@ -533,6 +537,20 @@ spec:
 {{ toYaml .Values.filebeat.resources | indent 10 }}
         terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod }}
       {{- end }}
+      {{- if .Values.metrics.enabled }}
```

Comment:
I don't understand the connection between `metrics.enabled` and `fluentd`. Why not introduce a separate flag for `fluentd`

Reply:
My thinking with bundling them together was that the all of the JFrog documentation has the two tightly bundled and your support team indicated this was the recommended way to expose metrics.  If you think I should split them up, it shouldn't be too much work.

Reply:
`/var/opt/jfrog/artifactory/log/` also has the logs. I think its better to use that. Same for the `JFROG_HOME`. I don't think you need anything else other than the log folder, right?

Reply:
@eldada @chukka WDYT? Would appreciate your take here

Reply:
In my opinion we should split them up, as not everyone will want to have fluentd if they just want prometheus metrics

Reply:
Generally I think what you are saying makes sense but I had a lot of back and forth with JFrog support about getting this working and was explicitly told, and your documentation seems to indicate this, that Fluentd was the supported and recommended way to expose Artifactory metrics.  I do agree that the chart should be flexible but I also think we should want to make it as frictionless as possible for developers to get metrics.  Maybe that means splitting them up and just making clearer documentation on this chart and your confluence pages

Reply:
Yes, I think that's reasonable. It would be great to have `fluentd.enabled` and `metrics.enabled` separately

Reply:
Pushed up a new commit decoupling these two.